UCSRest/src/main/java/it/cnr/iit/ucsrest/properties/UCSRestRequestManagerProperties.java

```diff
@@ -15,6 +15,9 @@
     @Value( "${api-remote-response}" )
     private String apiRemoteResponse;
 
+    @Value( "${active}" )
```

Comment:
Can we agree on a default mode? For example if the property is not defined in the file (forgotten) then it is defaults to true in the program like 
 @Value( "${active:true}" )